src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -2735,15 +2719,15 @@ internal abstract partial class BaseInvocationExpression : Operation, IHasArgume
         /// True if the invocation uses a virtual mechanism, and false otherwise.
         /// </summary>
         public bool IsVirtual { get; }
-        protected abstract ImmutableArray<IArgument> ArgumentsInEvaluationOrderImpl { get; }
+        protected abstract ImmutableArray<IArgument> ArgumentsImpl { get; }
         public override IEnumerable<IOperation> Children
         {
             get
             {
                 yield return Instance;
```

Comment:
Should `Instance` be skipped if `null`?

Reply:
@heejaechang is addressing this for all the operations in a separate PR